src/nodes/kolibri-broker/kolibri-broker.html/index.ts

```diff
@@ -0,0 +1,79 @@
+/*
+* Copyright 2021 HMS Industrial Networks AB
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http: //www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+
+import { EditorRED } from "node-red";
+import { KolibriBrokerEditorNodeCredentials, KolibriBrokerEditorNodeProperties } from "./modules/types";
+
+declare const RED: EditorRED;
+
+RED.nodes.registerType<KolibriBrokerEditorNodeProperties, KolibriBrokerEditorNodeCredentials>("kolibri-broker", {
```

Comment:
Why do we register two "kolibri-broker" ? (here and in kolibri-broker.ts)

Reply:
The first one is the client part will be merged into html file and is running on the browser. Second time is the runtime part which is executed on the node-red server.